### DISCUSSION TITLE
How to sync with Sqlite using change tracking

### DISCUSSION THREAD

**Discussion by AlfaGTA** [Mimetis/Dotmim.Sync#612] [Q&A] [answered]
Hi,
I trying to sync Sql server db using change tracking. As mentioned in the documentation, thanks to that, 'no more tracking tables and triggers in your database', but in Sqlite db I still see tracking tables.
Here is the code:
       
            `string connString = $@"Server={dataSource};Database={dbName};User Id=sa;Password=***";
            // Sql Server provider, the "server" or "hub".
            SqlSyncChangeTrackingProvider serverProvider = new SqlSyncChangeTrackingProvider(connString);

            // Sqlite Client provider acting as the "client"
            SqliteSyncProvider clientProvider = new SqliteSyncProvider("Vendo.db");

            // Tables involved in the sync process:
            var tables = new string[]
            {"Articoli", "DatiStorico", "Magazzini"};

            // Sync agent
            SyncAgent agent = new SyncAgent(clientProvider, serverProvider, tables);

            var result = await agent.SynchronizeAsync();`

I attach list of tables in Sqlite db:
![Immagine 2021-09-14 162148](https://user-images.githubusercontent.com/2462514/133275437-0a7a5f3e-42cc-4e64-aedc-db7921012a81.png)

Does it possible to sync from SqlServer to Sqlite using Change tracking?

**Comment by gaurakshay** [comment, marked as answer]
@AlfaGTA it is my understanding that the tracking tables will not be needed on the SQLServer if you use change tracking. However, since SQLite doesn't have a concept of change tracking, the tracking tables are still required to keep track of the changes in the SQLite DB. So what you are noticing is expected.

**Reply by Mimetis** [reply]
Indeed :) thanks @gaurakshay
